test/blackbox-tests/test-cases/glob_files_rec.t

```diff
@@ -82,3 +82,24 @@ Check that generated directories are ignored
 
   $ dune build @x
 
+Check that we get a nice error message if we pass and absolute path to `glob_files_rec`
+---------------------------------------------------------------------------------------
+
+Put $PWD in a file that can be read with the %{read:...} pform, so that the underline
+in the error message is of consisntent length on different systems.
+  $ printf $PWD > pwd
+
+  $ cat > dune <<EOF
+  > (rule
+  >  (alias x)
+  >  (deps (glob_files_rec %{read:pwd}/*.cmi))
+  >  (action (bash "for i in %{deps}; do echo \$i; done")))
```

Comment:
since we can't actually build the rule, we don't need `bash` here. How about just `(echo %{deps})` or something?